src/pages/Overview/index.js

```diff
@@ -1,16 +1,880 @@
+                <span className={styles.label}>{text[1]}</span>
+              </Text>
+            </div>
+          );
+        },
+      },
+      {
+        title: 'End Time',
+        dataIndex: 'end',
+        key: 'end',
+      },
+      {
+        title: '',
+        dataIndex: 'fav',
+        key: 'fav',
+        render: () => <Icon type="star" onClick={() => this.onFavourite()} />,
+      },
+      {
+        title: '',
+        dataIndex: 'action',
+        key: 'action',
+        render: (value, row) => {
+          return (
+            <div>
+              <EllipsisVIcon
+                onClick={() => this.showDrowpdown(`newrun${row.key}`)}
+                className="dropbtn"
+              />
+              <div className={styles.dropdown} id={`newrun${row.key}`} style={{ display: 'none' }}>
+                <div className={styles.dropdownContent}>
+                  <div className={styles.dropdownLink} onClick={() => this.markSeen(row)}>
+                    Mark as seen
+                  </div>
+                  <div className={styles.dropdownLink} onClick={() => this.deleteResult(row)}>
+                    Delete
+                  </div>
+                </div>
+              </div>
+            </div>
+          );
+        },
+      },
+    ];
+
+    const seenDataColumns = [
+      {
+        title: 'Result',
+        dataIndex: 'result',
+        key: 'result',
+        render: text => (
+          <div>
+            <Button variant="link" isInline style={{ marginBottom: '8px' }}>
+              {text[0]}
+            </Button>
+            <br />
+            <Text component={TextVariants.p} className={styles.subText}>
+              <span className={styles.label}>{text[1]}</span>
+            </Text>
+          </div>
+        ),
+      },
+      {
+        title: 'End Time',
+        dataIndex: 'end',
+        key: 'end',
+      },
+      {
+        title: 'Scheduled for deletion on',
+        dataIndex: 'deletion',
+        key: 'deletion',
+        render: text => {
+          const deleteDate = moment(new Date(Date.parse(text)));
+          const currDate = moment(new Date());
+          const remainingDays = deleteDate.diff(currDate, 'days');
+          if (remainingDays > 60) {
+            return (
+              <div>
+                <Text>
+                  {moment(text)
+                    .add(7, 'days')
+                    .format('YYYY-MM-DDTHH:mm:ss:SSSSSS')}
+                </Text>
+                <Progress
+                  min={0}
+                  max={expirationLimit}
+                  value={expirationLimit - remainingDays}
+                  size={ProgressSize.sm}
+                  measureLocation={ProgressMeasureLocation.none}
+                />
+              </div>
+            );
+          }
+          return (
+            <div>
+              <span>
+                {moment(text)
+                  .add(7, 'days')
+                  .format('YYYY-MM-DDTHH:mm:ss:SSSSSS')}
```

Comment:
Hey, we can use a helper function in utils.js

Reply:
Removed this. Didn't need formatting.